spec/system/managing_topics_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe 'managing topics' do
   before do
-    FactoryBot.create(:topic, name: 'What is your favourite colour')
+    @topic = FactoryBot.create(:topic, name: 'What is your favourite colour', id: 1)
```

Comment:
You should never set the database id yourself. This is for the database to manage its own internal integrity, and is best left to rails and the database.

Reply:
Don't declare instance variables inside rspec `before` blocks. The rspec way is to use `let` blocks to define variables that are accessible in tests.

```
let(:topic) { FactoryBot.create :topic, name: 'What is your favourite colour' }